scripts/generate_openapi_spec.py

```diff
@@ -35,8 +36,15 @@
 
 
 @click.command()
+@click.option(
+    '-d', '--dev',
```

Comment:
The flag could be of type `count` and not require a value like `True`, sth like `--publish` or `--update-in-commons`